Documentation/Tools/AssetSwapUtility.md

```diff
@@ -0,0 +1,21 @@
+# Asset swap utility
+
+Find and replace is ubiquitous when working in text and content creation tools. When you need to swap many assets within a Unity scene this is where the `AssetSwapUtility` (xref:Microsoft.MixedReality.Toolkit.Utilities.Editor.AssetSwapUtility) [ScriptableObject](https://docs.unity3d.com/Manual/class-ScriptableObject.html) and editor can lend a hand.
+
+The `AssetSwapUtility` saves all find and replace actions as a ScriptableObject so that it is trival to swap back and forth or save swap "themes" for future use.
+
```

Comment:
Would be helpful to include that this tool is going to be located in the `Microsoft.MixedReality.Toolkit.Unity.Tools` package.

Reply:
I like that idea, added!